cloudbuild.yaml

```diff
@@ -55,7 +57,13 @@ steps:
   - -c
   - cd e2e-tests;
     docker build -t ${_REPOSITORY}/e2e-tests:${_TAG} .;
-    docker build -t ${_REPOSITORY}/e2e-tests/controller-spark:${_TAG} ./controller-spark/.;
+    docker build -t ${_REPOSITORY}/e2e-tests/controller-spark:${_TAG} -t e2e-tests-controller-spark ./controller-spark/.;
```

Comment:
What is the benefit of the other tag `e2e-tests-controller-spark`?

Reply:
This was added for testing purposes, I have removed it in the latest commit. Thanks for letting me know.